Author  Pierre Ossman

[MMC] Support MMC version 4 cards
Version 4 of the MMC specification increased the version number of the
CID structure. None of the fields changed though so the only required
change is adding '4' to the approved list.

--- linux/drivers/mmc/mmc.c.orig	2006-02-04 20:51:52.000000000 +0300
+++ linux/drivers/mmc/mmc.c	2006-02-04 20:53:58.000000000 +0300
@@ -495,6 +495,7 @@
 
 		case 2: /* MMC v2.0 - v2.2 */
 		case 3: /* MMC v3.1 - v3.3 */
+		case 4: /* MMC v4 */
 			card->cid.manfid	= UNSTUFF_BITS(resp, 120, 8);
 			card->cid.oemid		= UNSTUFF_BITS(resp, 104, 16);
 			card->cid.prod_name[0]	= UNSTUFF_BITS(resp, 96, 8);
